### DISCUSSION TITLE
Host gltf files in public subdirectory

### DISCUSSION THREAD

**Discussion by ThatNerdUKnow** [pmndrs/gltfjsx#126] [Q&A] [answered]
Hi, I've been having some issues using this library. I generate a jsx element from my gltf file and add it to my project. I put my files in a directory `myrepo\public\models\blahblahblah\scene.gltf` I manually edit the imports in the generated gltf file to point to `/models/blahblahblah/scene.gltf` and I get an error Invalid character at position 0 in json. When I inspect the json sent to the client it appears to be valid json.

**Comment by drcmda** [comment, marked as answer]
this is not a library, it does not load or alter gltfs, it just extracts some text from it - that's all it does. if your THREE.GLTFLoader can't load a gltf or if the gltf can't access it's links (textures, etc) your issue is related to paths or broken files.

**Reply by eviltik** [reply]
My 2 cents : path prefix should be an optional parameter, so the generated JSX does not need to be edited manualy.
